src/main/java/commands/inputParser.java

```diff
@@ -0,0 +1,115 @@
+package commands;
+
+import exceptions.AragornException;
+
+public class inputParser {
```

Comment:
Class/enum names must be nouns and written in PascalCase. So in your case, it should be `InputParser` instead.